### DISCUSSION TITLE
Modify Projectile

### DISCUSSION THREAD

**Discussion by ghost** [ikemen-engine/Ikemen-GO#980] [Feature Request]
### Is your feature request related to a problem? Please describe.

Added here:https://github.com/ikemen-engine/Ikemen-GO/commit/9ef4d957b8293775f8d9a334dbd200473e642edc

Closing discussion

Similar to an explod and it having its own ModifyExplod, will be helpful to have fuller control of a projectile after it has been spawned with a new Modify Projectile sctrl. Less having to deal with making helper for simplistic modifications, and in discussion with Gacel on the topic of a "modify projectile" sctrl vs using a helper, helper uses way more resources, and having a simple modifyprojectile to switch simple things instead of relying on a full helper everytime can help streamline some things and use less resources depending on use case. 

### Describe the solution you'd like

Very similar to ModifyExplod, you have ModifyProjectile and you designate and ID, it follow same idea and conventions you see in modifyexplod and you can change a projectile based on ID after it has spawned. 

### Search terms

modify projectile

**Comment by LasombraDemon** [comment]
I was just going to suggest this one. This would allow for easier and more reasonable projectile reflection systems, too.
